_includes/header.html

```diff
@@ -174,3 +210,8 @@ <h2 class="element-invisible">Search form</h2>
     </div>
   </div>
 </div>
+<script type="text/javascript">
```

Comment:
Javascript should exist in the included JS file.

Reply:
Further, this should be handled by Bootstrap.

Reply:
ok